ui-components/src/helpers/filters.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  ListingFilterParams,
+  ListingFilterKeys,
+  EnumListingFilterParamsComparison,
+} from "@bloom-housing/backend-core/types"
+import { ParsedUrlQuery } from "querystring"
+
+function getComparisonForFilter(filterKey: ListingFilterKeys) {
+  switch (filterKey) {
+    case ListingFilterKeys.name:
+      return EnumListingFilterParamsComparison["="]
+    case ListingFilterKeys.neighborhood:
+      return EnumListingFilterParamsComparison["="]
+    case ListingFilterKeys.status:
+      return EnumListingFilterParamsComparison["="]
+  }
```

Comment:
Adding an exhaustive check to the switch statement lets us use the type system to make sure all filter types are handled, like
```suggestion
      default:
        const _exhaustiveCheck: never = filterKey;
        return _exhaustiveCheck;
    }
```
More here:
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#exhaustiveness-checking